src/Tvheadend.h

```diff
@@ -273,9 +274,9 @@ class CHTSPDemuxer
   void SetStreamingProfile(const std::string &profile);
 
 private:
-  P8PLATFORM::CMutex                        m_mutex;
+  mutable P8PLATFORM::CMutex              m_mutex;
```

Comment:
If you drop the lock you can drop this as well, I'll be re-introducing it in my PR.